### DISCUSSION TITLE
Uninstalled Azuracast, how to recover media files

### DISCUSSION THREAD

**Discussion by sb-200** [AzuraCast/AzuraCast#5742] [Q&A]
Hi, hoping someone may be able to help me

I uninstalled Azuracast due to an issue, however I believe the media files were retained. Is there any way these can be recovered via SFTP? I can't seem to find the folder they are located.

Thanks so much!

**Comment by Moonbase59** [comment]
SFTP might be difficult, since it is set up _inside_ the docker container.

But as long as you don’t have also uninstalled Docker, you just _might_ be lucky if you ssh into your host and check out `/var/lib/docker/volumes/azuracast_station_data/_data/(station_name)/media`.
